.travis.yml

```diff
@@ -33,8 +33,8 @@ env:
   - 
   - BACKEND="OPENMP"
 #  - BACKEND="PTHREAD"
-  - CMAKE_BUILD_TYPE=Debug COVERAGE=yes
-  - CMAKE_BUILD_TYPE=Debug BACKEND="OPENMP" COVERAGE=yes
+  - CMAKE_BUILD_TYPE=Debug COVERAGE=yes GTEST_FILTER="-*DeathTest*"
+  - CMAKE_BUILD_TYPE=Debug BACKEND="OPENMP" COVERAGE=yes GTEST_FILTER="-*DeathTest*"
```

Comment:
You are filtering out the DeathTest beyond the two builds that were failing.  Just making sure you realize that.